getonescriptspan_test.cc is just a test file that shouldn't be linked into any program.

Ref: https://github.com/google/cld3/pull/78
---
--- a/cmake/external/cld3/CMakeLists.txt	2022-12-31 07:19:02.000000000 +0100
+++ -	2023-01-06 13:30:13.082990927 +0100
@@ -80,7 +80,6 @@
     script_span/generated_entities.cc
     script_span/getonescriptspan.cc
     script_span/getonescriptspan.h
-    script_span/getonescriptspan_test.cc
     script_span/utf8statetable.cc
     script_span/offsetmap.cc
     script_span/text_processing.cc
